src/styles/main.scss

```
@@ -1,3 +1,87 @@
-body {
-  margin: 0;
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  &__body {
+    margin: 0;
+  }
+}
+
+$col-count: 7;
+$col-gap: $col-count - 1;
+$cell-size: 100px;
+$padding: 10px;
+
+@mixin square($size) {
+  width: $size;
+  height: $size;
+}
+
+@function getWidth($count, $size, $padding, $gap) {
+  @return $count * $size + $padding * 2 + $gap;
+}
+
+.calendar {
+  display: flex;
+  width: 100vw;
+  height: 100vh;
+}
+
+.month {
+  max-width: getWidth($col-count, $cell-size, $padding, $col-gap);
+  margin: auto;
+  padding: $padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    @include square($cell-size);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: 0.5s;
+
+    @for $day from 1 through 31 {
+      &:nth-child(#{$day})::before {
+        content: "#{$day}";
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+      background-color: pink;
+    }
+  }
+}
+
+$week-days:
+  "mon" 1,
```

```suggestion
  "mon" 0,
```
Начинай с нуля, чтобы не делать `($number - 1)`

---

тоді потрібно буде додовати одиницю на 78 рядку, щоб врахувати 1px gap